packages/mobile/src/account/AccountKeyEducation.tsx

```diff
@@ -16,7 +16,7 @@ export default function AccountKeyEducation(props: Props) {
     if (props.route.params?.nextScreen) {
       navigate(props.route.params?.nextScreen)
     } else {
-      navigateProtected(Screens.BackupPhrase)
```

Comment:
Should we remove navigateProtected now?

Reply:
Wow, man. I've just realised it was the only place it was used... Will remove it, yeah!